Improve extension signature verification section (#8267)
microsoft/vscode-docs: docs/configure/extensions/extension-marketplace.md

The Visual Studio Marketplace signs all extensions when they are published. VS Code verifies this signature when you install an extension to check the integrity and the source of the extension package.

> When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`. This error can be caused by a variety of reasons and should you encounter this error, exercise caution before deciding to install anyway. If you want to disable extension signature verification, you can use the `setting(extensions.verifySignature)` setting.

Reviewer: ```suggestion
> [!IMPORTANT]
> When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`. This error can be caused by a variety of reasons and should you encounter this error, exercise caution before deciding to install anyway. Disable extension signature verification with the `setting(extensions.verifySignature)` setting.
```

> #### Package Integrity Issues

Reviewer: ```suggestion
#### Package integrity issues
```

For package integrity issues, it's recommended that you contact the [Visual Studio Marketplace team](mailto:vsmarketplace@microsoft.com?subject=Extension%20Signature%20Verification%20Issue) to report the issue. Make sure to include the extension ID.

> #### Other Issues

Reviewer: ```suggestion
#### Other issues
```

> For other issues like unsupported environment or unknown reasons, it's recommended that you [report an issue](https://github.com/microsoft/vscode/issues/new) with VS Code by providing necessary information and including the following log - `F1 > Open View... > Shared`.

Reviewer: ```suggestion
For other issues like an unsupported environment or unknown reasons, it's recommended that you [report an issue](https://github.com/microsoft/vscode/issues/new) with VS Code by providing all necessary information and including the shared log: `kb(workbench.action.showCommands)` > **Open View...** > **Shared**.
```
